hub3/fragments/api.go

```diff
@@ -336,6 +336,25 @@ func NewSearchRequest(params url.Values) (*SearchRequest, error) {
 			}
 
 			tree.PageSize = int32(hint)
+		case "start":
+			start, err := strconv.Atoi(params.Get(p))
+			if err != nil {
+				log.Printf("unable to convert %v to int for %s", v, p)
+				return sr, err
+			}
+			sr.Start = int32(start)
+		case "searchAfter":
+			var sa = make([]interface{}, 0)
+			parts := strings.Split(params.Get(p), ",")
```

Comment:
The comma as string delimiter is poorly chosen. There are inventory numbers with a comma. Perhaps a pipe?

Reply:
The comma is selected by ElasticSearch. We take the exact string form as returned from the elasticsearch response.